modules/nf-core/taxpasta/merge/main.nf

```diff
@@ -0,0 +1,38 @@
+process TAXPASTA_MERGE {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "bioconda::taxpasta=0.1.1"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/taxpasta:0.1.1--pyhdfd78af_0':
+        'quay.io/biocontainers/taxpasta:0.1.1--pyhdfd78af_0' }"
+
+
+    input:
+    tuple val(meta), path(profile)
+    path samplesheet
```

Comment:
This needs to be inserted into the command with `-s` if provided.

@Midnighter is this also mututally exclusive to the actual profiles?

e.g.

```
script:
input = samplesheet ? '-s ${samplesheet}' : '$profiles'

"""
taxpasta \\
    $args \\
    $input 
"""
```

Reply:
You could do that but you don't have to. When a sample sheet is provided, it will simply ignore any extra profiles provided. So it is mutually exclusive but doesn't cause an error.